source/transactions-convenient-api/transactions-convenient-api.rst

```diff
@@ -381,14 +388,14 @@ their users to design idempotent callbacks.
 
 .. _Retry Transactions and Commit Operation: https://docs.mongodb.com/manual/core/transactions/#retry-transaction-and-commit-operation
 
-The commit is retried after a write concern timeout error
----------------------------------------------------------
+The commit is retried after a write concern wtimeout error
```

Comment:
Was this done for clarity's sake? IMO, a "write concern timeout" implies wtimeout. "write concern wtimeout" reads a bit strange, though. Perhaps "write concern timeout (i.e. wtimeout)"? Some text below still uses the phrase "write concern timeout" so I think the single "i.e." addition here would mesh nicely (similar to defining an acronym on its first use).

Reply:
Yeah, I changed it for clarity because now we are considering two different write concern timeouts. wtimeout:
`{ok:1, writeConcernError: {code: 64, codeName: "WriteConcernFailed", errmsg: "waiting for replication timed out", errInfo: {wtimeout: True}}}`
And maxTimeMS:
`{ok:1, writeConcernError: {code: 50, codeName: "MaxTimeMSExpired"}}`

Updated the title with "write concern timeout (i.e. wtimeout)".